python/cudf/cudf/_lib/strings/translate.pyx

```diff
@@ -30,10 +30,14 @@ def translate(Column source_strings,
 
     for key in mapping_table:
         value = mapping_table[key]
+        if type(value) is int:
+            value = chr(value)
         if type(value) is str:
-            value = ord(value)
+            value = int.from_bytes(value.encode(), byteorder='big')
+        if type(key) is int:
+            key = chr(key)
         if type(key) is str:
-            key = ord(key)
+            key = int.from_bytes(key.encode(), byteorder='big')
```

Comment:
Are these conversions we expect to do in other places as well? If so, it may be worth making little utility functions for them. If not, this looks fine

Reply:
This is the only place I know that works with passing individual characters to/from Python like this. All of the other cases pass strings (or string-scalar) even if only a single character is passed. The Cython code uses the `encode()` function to convert these to UTF-8.

Reply:
👍